protostar/commands/test/test_execution_environment.py

```diff
@@ -45,9 +46,11 @@ class TestExecutionEnvironment:
     def __init__(self, include_paths: List[str]):
         self.starknet = None
         self.test_contract: Optional[StarknetContract] = None
+        self.tmp_state = TmpState()
         self._expected_error: Optional[RevertableException] = None
         self._include_paths = include_paths
         self._test_finish_hooks: Set[Callable[[], None]] = set()
+        self._hijacked_hint_locals: Optional[Dict[str, Any]] = {}
```

Comment:
Why this is optional if we set it here?

Reply:
I am not sure why do we need this at all

Reply:
removed